master/lustrefactory.py

```diff
@@ -56,6 +59,65 @@ def buildzfsCommand(props):
     return args
 
 @util.renderer
+def configureCmd(props):
+    args = ["./configure"]
+    style = props.getProperty('buildstyle')
+
+    if style == "deb" or style == "rpm":
+        with_zfs = props.getProperty('withzfs')
+        if with_zfs and with_zfs == "yes":
+            args.extend(["--with-zfs"])
+        else:
+            args.extend(["--without-zfs"])
+
+        with_ldiskfs = props.getProperty('withldiskfs')
+        if with_ldiskfs and with_ldiskfs == "yes":
+            args.extend(["--enable-ldiskfs"])
+        else:
+            args.extend(["--disable-ldiskfs"])
+    elif style == "srpm":
+        args.extend(["--enable-dist"])
+
+    return args
+
+@util.renderer
+def makeCmd(props):
+    args = ["sh", "-c"]
+    style = props.getProperty('buildstyle')
+
+    if style == "srpm":
+        args.extend(["make -j$(nproc) srpm"])
+    elif style == "deb":
+        args.extend(["make -j$(nproc) debs"])
+    elif style == "rpm":
+        args.extend(["make -j$(nproc) rpms"])
+    else:
+        args.extend(["make -j$(nproc)"])
+
+    return args
+
+@util.renderer
+def rpmbuildCmd(props):
```

Comment:
This should really be removed.  We should just do "make rpms" on the RHEL-like systems.